plugins/org.jboss.tools.windup.ui/html/windup.html

```diff
@@ -4,5 +4,169 @@
 </head>
 <body>
 </body>
-    <h4>Windup - Getting Started</h4>
+    <h1>Getting Started with the Windup Eclipse Plugin</h2>
+
+    <h2>Identify and Resolve Migration Issues with Windup</h2>
+    <div>
+      <p>
+        Run Windup against your project to identify and fix migration issues.
+      </p>
+      <ol>
+        <li>
+          Import the project to analyze.
+        </li>
+        <li>
+          <p>
+            <a href="#create-config">Create a Windup run configuration</a>. From the Issue Explorer, press the Windup button (<img src="../icons/windup.png"/>).
+          </p>
+          <p>
+            <img src="../images/windup_button_create_config.png"/>
+          </p>
+          <p>
+            At a minimum, select the project to analyze. Set additional options as needed.
+          </p>
+        </li>
+        <li>
+          <p>
+          Click <b>Run</b> to execute Windup.
+          </p>
+        </li>
+        <li>
+          <p>
+            <a href="#review-issues">Review Windup issues</a> listed in the Issue Explorer.
+          </p>
+        </li>
+        <li>
+          <p>
+            <a href="#resolve-issues">Resolve Windup issues</a> by manually updating code or by using quick fixes when available.
+          </p>
+        </li>
+        <li>
+          <p>
+            Run Windup again as necessary. Use the drop down next to the Run button (<img src="../icons/run_exc.png"/>) to run an existing configuration.
+          </p>
+        </li>
+      </ol>
+    </div>
+
+    <a name="create-config"><h2>Create a Windup Run Configuration</h2></a>
+    <div>
+      <p>
+        A Windup run configuration can be created using the Windup button (<img src="../icons/windup.png"/>). You can create multiple run configurations, each with a unique name.
+      </p>
+      <h3>Input</h3>
+      <p>
+        Select a <b>migration path</b>. This determines which Windup rulesets are used. The migration path defaults to <b>Anything to EAP 7</b>, but can target <b>EAP 6</b> or specify a particular source.
```

Comment:
This has changed a little, we discussed this some and @johnsteele has taken out the "Anything to EAP 7" item since we don't have any WLS/WAS->EAP 7 rules -- right now the options are only {Anything,WLS,WAS} to EAP 6.

I think we need to also add the EAP 5,6->7 paths to that dialog since those are supported, and discuss the default (I prefer defaulting to something that targets 7 over 6, YMMV), then we can come back and edit this line.